CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 gz_configure_project(
-  REPLACE_IGNITION_INCLUDE_PATH gz/math
+  REPLACE_INCLUDE_PATH gz/math
```

Comment:
I think this line can be removed since the default path is `gz/math` as of https://github.com/gazebosim/gz-cmake/pull/277

Reply:
Fixed here and everywhere else https://github.com/gazebosim/gz-math/pull/462/commits/008784ae83b8c179317df2a18ec26a7c1bfcabc7